internal/gatewayapi/testdata/gateway-with-listener-with-tcproute-with-mismatch-port-protocol.out.yaml

```diff
@@ -72,8 +72,7 @@ tcpRoutes:
     parents:
     - conditions:
       - lastTransitionTime: null
-        message: 'backend Service validation failed: TCP Port 8081 not found on Service
-          default/service-1'
+        message: TCP Port 8081 not found on Service default/service-1
```

Comment:
Will handle this in a follow-up PR, and use the same message format as HTTPRoute.